pyomo/contrib/mindtpy/cut_generation.py

```diff
@@ -184,3 +245,75 @@ def add_int_cut(var_values, solve_data, config, feasible=False):
     #     MindtPy.MindtPy_linear_cuts.integer_cuts.add(expr=int_cut)
     # else:
     #     MindtPy.MindtPy_linear_cuts.feasible_integer_cuts.add(expr=int_cut)
+
+
+def add_affine_cuts(solve_data, config):
```

Comment:
I'm not really sure what this function does.

Reply:
This function is to add **McCormick Relaxation cuts** when the original problem is a nonconvex MINLP. The cuts here derives from [MC++](https://omega-icl.github.io/mcpp/page_MCCORMICK.html). We will add the documentation for all the functions in MindtPy as soon as possible.